dashboards/grafana-dashboard-rhtap-slos.configmap.yaml

```diff
@@ -1299,60 +1153,49 @@ data:
                 "uid": "${datasource}"
               },
               "editorMode": "code",
-              "expr": "sum(openshift_route_status{namespace=\"gitops-service-argocd\", status=\"True\"})",
-              "hide": true,
+              "expr": "1-sum(increase(has_application_failed_creation_total[$__range]))\n/\nsum(increase(has_application_creation_total[$__range]))",
               "legendFormat": "__auto",
               "range": true,
               "refId": "A"
-            },
-            {
-              "datasource": {
-                "type": "prometheus",
-                "uid": "${datasource}"
-              },
-              "editorMode": "code",
-              "expr": "sum(openshift_route_status{namespace=\"gitops-service-argocd\"})",
-              "hide": true,
-              "legendFormat": "__auto",
-              "range": true,
-              "refId": "B"
-            },
-            {
-              "datasource": {
-                "name": "Expression",
-                "type": "__expr__",
-                "uid": "__expr__"
-              },
-              "expression": "$A / $B * 100",
-              "hide": false,
-              "refId": "% Healthy",
-              "type": "math"
             }
           ],
           "title": "Measured",
-          "type": "timeseries"
+          "type": "gauge"
+        },
+        {
+          "collapsed": false,
+          "gridPos": {
+            "h": 1,
+            "w": 24,
+            "x": 0,
+            "y": 52
+          },
+          "id": 35,
+          "panels": [],
+          "title": "Application Deletion SLO",
```

Comment:
What do you think about putting all the HAS SLOs under one row titled `Application Service SLOs`?

Reply:
Done 
<img width="1009" alt="image" src="https://github.com/redhat-appstudio/o11y/assets/84398375/1c2983fd-b148-4c1b-bb22-26b89d36168d">